homeassistant/components/light/blinkt.py

```diff
@@ -37,19 +37,22 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     name = config.get(CONF_NAME)
 
-    add_devices([BlinktLight(blinkt, name)])
+    add_devices([
+        BlinktLight(blinkt, name, index) for index in range(blinkt.NUM_PIXELS)
+    ])
 
 
 class BlinktLight(Light):
     """Representation of a Blinkt! Light."""
 
-    def __init__(self, blinkt, name):
+    def __init__(self, blinkt, name, index):
         """Initialize a Blinkt Light.
 
         Default brightness and white color.
         """
         self._blinkt = blinkt
-        self._name = name
+        self._name = "{}_{}".format(name, index)
```

Comment:
This seems to be a breaking change, i.e. existing automations will need modification – right?

I am not sure how these lights are used, is it relevant to maintain the ability to set all pixels at once?

Reply:
Yes, it is indeed a breaking change.

As for ability to set all at once, I've consulted it on #devs:
```
matemaciek - 14/11/2017
If so, there are some decisions to be made regarding controlling all vs controlling single one. Some possibilities:
- remove _all light
- introduce option mode in configuration, with valid values separated and together, and show 8 and 1 control respectively
- try to support both modes at the same time
The last option is the hardest, as now this component is very simple and does not poll for changes. There is also a problem with colour of _all light when separated lights have different colours.

OverloadUT - 14/11/2017
For controlling multiple, I believe you should create a scene component
Each light is its own light entity, and then the user can define scenes which group lights

balloob - 14/11/2017
There is already a scene component that can control lights
```

These are 8 RGB LEDs, which are perfect for e.g. notifying state of things, just like lights on a router etc. They of course can be used as one big lamp, but such use is IMHO so non-standard, that it would be reasonable to create group / scene for all lights.

Reply:
Cool, that makes sense. It's good to have this information easily available when users see in the changelog that a breaking change has happened. So please add a single "breaking change notice" sentence to the top of this PR describing the entity name change and scene/group workaround and then we can merge it.

Reply:
OK, I did it.